packages/amplication-client/src/Project/ProjectSideBar.scss

```diff
@@ -0,0 +1,42 @@
+@import "../style/index.scss";
+
+.side-bar-layout {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 16px;
+  width: 280px;
+  height: 906px;
+  background: #15192c;
+  border: 1px solid #373d57;
+  border-radius: 8px;
+
+  &__label {
+    width: 248px;
+    height: 18px;
+    font-family: "Poppins";
+    font-style: normal;
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    display: flex;
+    align-items: center;
+    color: #a3a8b8;
+  }
+
+  &__inner {
+    height: 688px;
+    width: 248px;
+    border-top: 1px solid #2c3249;
+    border-bottom: 1px solid #2c3249;
+    padding: 16px 0;
+  }
+
+  &__tabs {
```

Comment:
please use variables and mixins

Reply:
if you need, you can wait to merge with @barshimi before setting the style of this component (until then - just leave it empty)